### PR TITLE
Update ColorPicker.js

### PR DESCRIPTION
**Pull request by KinkGD**
_Original "wrong" PR i've closed : #915_ 

This "fix" "correct" the issue #896 

image

i use double quotes on "fix" and "correct" because it's a bit "hacky" :)

### CODE REVIEW

**Comment by 4ian** [comment]
Unfortunately, switching to absolute make the color picker display in the parent div which has a `position: relative`, and so it's cut off in some smaller dialogs
image

So I think we need to stay with fixed.
Even with `fixed`, when using `right`, you specify the margin to the right of the parent div that has a `position: relative` I think, so we end up with this:

image

I think we need to work with something that just move the position of the popover div. Something like `transform: "translateX(-174px)"`. (174px because width of button = 46, width of color picker = 220)

**Comment by KinkGD** [comment]
Ok, that i've tested and it works on both screen (action editor & changing the color of the scene, wich i didn't tested in the first commit).

My second commit is exactly what you were advising, i hope this time it will be ok :)

**Comment by 4ian** [comment]
Almost there, checks are failing because the file is not automatically format. Can you run `npm run format`/(or `yarn format`) in the newIDE/app folder? Thanks!

**Comment by 4ian** [comment]
GitHub strangely show the whole ColorPicker.js file as being changed.. that's really strange. What is the git client that you're using? Anyway, formatting test is passing on SemaphoreCI so I think it's good to go :)

**Comment by KinkGD** [comment]
I submitted the file directly through github the website, like a i did for my other contribution, and after i ran the format command.

i'll try to use gitkraken in the future, i've just installed it.

**Comment by 4ian** [comment]
Thanks for working on this! 👍

**Comment by blurymind** [comment]
GitHub sometimes does this when you edit the file online. I use gitkracken
too, it's fantastic

**Comment by zatsme** [comment]
> GitHub sometimes does this when you edit the file online. I use gitkracken
too, it's fantastic

I use this, so easy and just works!

File: newIDE/app/src/UI/ColorField/ColorPicker.js [hunk 1]
```diff
@@ -1,115 +1,116 @@
...
+    background: '#fff',
+    borderRadius: '1px',
+    boxShadow: '0 0 0 1px rgba(0,0,0,.1)',
+    display: 'inline-block',
+    cursor: 'pointer',
+  },
+  popover: {
+    position: 'fixed',
+    zIndex: '2',
+    transform: 'translateX(-174px)',
```
Comment (4ian) [inline newIDE/app/src/UI/ColorField/ColorPicker.js]: For the record, adding `transform` here created a new "container" (see section about fixed on MDN: https://developer.mozilla.org/en-US/docs/Web/CSS/position) so the cover div, that is supposed to detect clicks outside of the picker, was not properly filling the dialog/screen.
I'm searching how to fix that.

Comment (4ian) [reply newIDE/app/src/UI/ColorField/ColorPicker.js]: Fixed in 69978f9681a640842fb74ce9a000d1c6cce9a6a8 :)

**Pull request merged**